### PR TITLE
Update 4131 documentation

### PR DESCRIPTION
**Pull request by Xevib**

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

File: docs/distri/adm-pub/cnmc.md [hunk 1]
```diff
@@ -987,4 +988,860 @@ Número de lecturas any circular  | Número de lectures d'activa efectuadas en l
 Camp            | Descripció
 :---------------|:----------------------------------------------
 Número de sèrie | Número de sèrie del comptador
-CINI            | Codi CINI del comptador
\ No newline at end of file
+CINI            | Codi CINI del comptador
+
+## Inventaria 4131
```
Comment (ecarreras) [inline docs/distri/adm-pub/cnmc.md]: Hi ha un error tipogràfic i fa que no funcioni l'enllaç de dalt de la pàgina

File: docs/distri/adm-pub/cnmc.md [hunk 2]
```diff
@@ -987,4 +988,860 @@ Número de lecturas any circular  | Número de lectures d'activa efectuadas en l
...
+
+
+.. note::
+    Relacionats amb aquest módul hi ha el módul de subestacions, cel·les i elements de tall, despatxos, posicions,
+    Catàleg de cables AT, Catàleg de cables BT, Expedients i Condensadors.
+
+.. note::
+    S'han de unificar els CINI's als publicats a la resolució 4131/2016
+
+.. note::
```
Comment (ecarreras) [inline docs/distri/adm-pub/cnmc.md]: No funcionen els `.. note::` això és en `.rst` en markdown ho has de fer amb `!!! note:` see an [example](https://github.com/gisce/powerp-docs/blame/master/docs/base/atr.md#L193)

File: docs/distri/adm-pub/cnmc.md [hunk 3]
```diff
@@ -987,4 +988,860 @@ Número de lecturas any circular  | Número de lectures d'activa efectuadas en l
...
+   El camps que s'usara com a Tipus d'instal·lacio es el Tipologia CNMC(desplegable)
+
+### Carrega de fitxers de la 4771
+
+
+Per tal que el ERP pugui fer el calculdel camp estat de la 4131 s'h de carregar el fitxer presentat en la 4771.
+El format del ftixer que s'ha de carregar es format ZIP en el cual a l'arrel del fitxer hi hauran els 8 fitxers entregats.
+
+.. image:: _static/carrega_4771.png
```
Comment (ecarreras) [inline docs/distri/adm-pub/cnmc.md]: Use markdown format

File: docs/distri/adm-pub/cnmc.md [hunk 4]
```diff
@@ -987,4 +988,860 @@ Número de lecturas any circular  | Número de lectures d'activa efectuadas en l
...
+
+.. note::
+    S'han de comprovar tots els registres que tenen data_APM de l'any en que es genera el fitxer (en aquest cas 2016)
+    perque no sortiran als fitxers .csv.
+    S'han de comprovar les instal·lacions **reformades** durant l'any en que es genera el fitxer i comprovar les 
+    dates_APM dels registres en questió per veure si han de sortir o no als .csv y si cal modificar la data_APM per
+    que apareixin en la generació dels fitxers.
+
+ORÍGEN
+^^^^^^
```
Comment (ecarreras) [inline docs/distri/adm-pub/cnmc.md]: Use markdown format

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Comment by jaumef** [comment]
From test output:

> _There are missing strings in the po file!_

Before adding them, solve the conflicts on messages.pot and messages.po. Then use the following commands (From README.md):

> _cd powerp-docs_
> _mkvirtualenv powerp-docs_
> _pip install -r requirements.txt_
> _export PYTHONPATH=$PWD/sitecustomize_
> _mkdocs build -f mkdocs_es.yml_
> _msgmerge -U locales/lang/LC_MESSAGES/messages.po locales/messages.pot_
> _poedit locales/lang/LC_MESSAGES/messages.po_

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_cnmc_resolucio_4131/

**Pull request merged**